kie-pmml-bom/pom.xml

```diff
@@ -193,6 +194,19 @@
         <classifier>tests</classifier>
         <version>${version.org.kie}</version>
       </dependency>
+      <!-- Evaluator -->
+      <dependency>
+        <groupId>org.kie</groupId>
+        <artifactId>kie-pmml-evaluator-core</artifactId>
+        <classifier>tests</classifier>
+        <version>${version.org.kie}</version>
+      </dependency>
+      <!-- Tests -->
```

Comment:
@gitgabrio @mareknovotny I'm not convinced about this one. Do we want to add versions of test modules into bom for users?

Reply:
it depends if this is for end users to consume in their applications.
@gitgabrio this and below GAV for internal testing or for end user consumption ?
you can have it added in kie-parent if that is only internal dependencyManagement

Reply:
@mareknovotny @jiripetrlik @danielezonca 
One of the reason behind the architecture I designed for PMML it is to allow the "end users" to write their own model implementation(s), overriding/replacing our(s).
From this point of view, even this module should be considered available for end users.
Does this make sense ?

Reply:
@danielezonca @jiripetrlik 
Hi! Could we proceed with this one? It is needed by other ones currently opened and one way or another I would like to close it

Reply:
Fine for me to expose this module if you expect other user can need it

Reply:
@gitgabrio Understand, I'm fine with merging this.